source/DataAccess/IDDLHelper.ts

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
This assumes integrations will work off of objects and queries in string form to get 'ddl'. I initially went some generics here but that quickly got weird and felt like a solution looking for a problem we don't yet have. For now it's an interface to things that can be queried using simple statements.